### PR TITLE
[backend] update pb and command args

### PR DESCRIPTION
**Pull request by fenrir-z**
* command args
  * seg -> sem-seg
  * det-box -> det
* pb
  * OT_DET_BOX -> OT_DET
  * OT_SEG -> OT_SEM_SEG
  * ExportFormat -> ImportExportFormat (will also use in command import dataset, so add Import here)
  * remove IEF_LS_JSON

### CODE REVIEW

File: ymir/backend/src/common/id_definition/state.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ class ResultState(IntEnum):
 class ObjectType(IntEnum):
     unknown = mir_cmd_pb.OT_UNKNOWN
     classification = mir_cmd_pb.OT_CLASS
-    object_detect = mir_cmd_pb.OT_DET_BOX
-    segmentation = mir_cmd_pb.OT_SEG
+    object_detect = mir_cmd_pb.OT_DET
+    segmentation = mir_cmd_pb.OT_SEM_SEG
```
Comment (phoenix-xhuang) [inline ymir/backend/src/common/id_definition/state.py]: semantic_segmentation

File: ymir/backend/src/ymir_app/app/constants/state.py [hunk 2]
```diff
@@ -87,8 +87,8 @@ class MiningStrategy(IntEnum):
 class ObjectType(IntEnum):
     unknown = mir_cmd_pb.ObjectType.OT_UNKNOWN  # 0
     classification = mir_cmd_pb.ObjectType.OT_CLASS  # 1
-    object_detect = mir_cmd_pb.ObjectType.OT_DET_BOX  # 2
-    segmentation = mir_cmd_pb.ObjectType.OT_SEG  # 3
+    object_detect = mir_cmd_pb.ObjectType.OT_DET  # 2
+    segmentation = mir_cmd_pb.ObjectType.OT_SEM_SEG  # 3
```
Comment (phoenix-xhuang) [inline ymir/backend/src/ymir_app/app/constants/state.py]: 为啥没改？

File: ymir/command/proto/mir_command.proto [hunk 3]
```diff
@@ -138,17 +138,18 @@ enum ObjectType {
...
     OST_SEG_POLYGON = 31;
 }
 
-enum ExportFormat {
-    EF_NO_ANNOTATIONS = 0;
-    EF_VOC_XML = 1;
-    EF_ARK_TXT = 2;
-    EF_LS_JSON = 3;
-    EF_COCO_JSON = 4;
+enum ImportExportFormat {
```
Comment (phoenix-xhuang) [inline ymir/command/proto/mir_command.proto]: 脱离了语境谁知道importexportformat是什么意思

这里是AnnoFormat

Comment (fenrir-z) [reply ymir/command/proto/mir_command.proto]: done

File: ymir/backend/src/ymir_app/app/constants/state.py (hunk 2 above)
Comment (fenrir-z) [reply ymir/backend/src/ymir_app/app/constants/state.py]: done

File: ymir/backend/src/common/id_definition/state.py (hunk 1 above)
Comment (fenrir-z) [reply ymir/backend/src/common/id_definition/state.py]: done

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**